packages/connect-form/src/components/advanced-options-tabs/ssh-tunnel-tab/ssh-tunnel-password.tsx

```diff
@@ -0,0 +1,92 @@
+import { ConnectionOptions } from 'mongodb-data-service';
+import React, { ChangeEvent } from 'react';
+import { css } from '@emotion/css';
+import { TextInput, spacing } from '@mongodb-js/compass-components';
+import { SSHConnectionOptions } from '../../../hooks/connection-options-handler';
+
+const inputFieldStyles = css({
+  width: '50%',
```

Comment:
Seems like a better place to set the size here would be some sort of wrapper for the whole ssh tab instead of doing it here. But it also a bit weird to me that we span general tab to the whole size of the form, but keep this one on 50% for some reason